### PR TITLE
NECO-197 fixed to refresh token

### PR DESCRIPTION
**Pull request by hishida**
## Ticket

[NECO-197](https://pflabo.atlassian.net/browse/NECO-197)
## Changes
- Fixed to refresh token when token will soon expire.
## Tests
- Tested to refresh token automatically when initialize `RicohAPI::MStorage::Client` with `RicohAPI::Auth::Client` instance.

Details and test results are in [ticket](https://pflabo.atlassian.net/browse/NECO-197)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,12 @@ access_token = api_session.access_token
 ### Constructor
 
 ```ruby
+mstorage = RicohAPI::MStorage::Client.new access_token, {auth_client: client}
```
Comment (sohda) [inline README.md]: `{}` is unnecessary.

File: lib/ricohapi/mstorage/client.rb [hunk 2]
```diff
@@ -120,6 +125,10 @@ def remove_meta(media_id, key)
       private
 
       def handle_response(as_raw = false, retrying = false)
+        if self.auth_client
+          self.token = self.auth_client.api_token_for! RicohAPI::MStorage::SCOPE
+        end
```
Comment (sohda) [inline lib/ricohapi/mstorage/client.rb]: I think this if-statement can be inline style.
ex) https://github.com/hishida/media-storage-rb/blob/cddfc0e84c0c0a83557e202b13cdbee93177aef4/lib/ricohapi/mstorage/client.rb#L139

File: README.md (hunk 1 above)
Comment (hishida) [inline README.md]: Thank you for your pointing out. I fixed it.

File: lib/ricohapi/mstorage/client.rb (hunk 2 above)
Comment (hishida) [inline lib/ricohapi/mstorage/client.rb]: Surely. I fixed it.

**Pull request merged**